compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEConvertCoreForallToFor.cpp

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file the conversion of `scf.forall` into `scf.for` with `aie.core`
```

Comment:
nit: This file converts?

Reply:
Oopsie, had automerge on and it just merged before I clicked to disable..

Reply:
No worries!